main.go

```diff
@@ -26,13 +27,14 @@ func (m *GreenReviewsTooling) BenchmarkPipeline(ctx context.Context,
 	version,
 	benchmarkJobURL,
 	kubeconfig string,
+	prometheus_url string,
```

Comment:
```suggestion
	prometheusURL string,
```

We should use 🐫 case